MaterialDesignThemes.Wpf.ThemeCreator/MaterialDesignThemes.Wpf.ThemeCreator.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
I believe this project can be removed now

Reply:
Actually, I don't know whi git didn't removed. On my solution explorer it don't list anymore:

![image](https://user-images.githubusercontent.com/15158923/92303210-1c059380-ef49-11ea-9aee-f56cbb11d301.png)

I'll try to remove direcly from File Explorer.

Reply:
"I'll try to remove direcly from File Explorer."

It works! 😁